src/main/kotlin/lotto/domain/Lottos.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.domain
+
+class Lottos(_values: List<Lotto>) {
+    var values: List<Lotto> = _values
+        private set
+
+    val size: Int
+        get() = values.size
+
+    fun compareAllTo(lotto: Lotto) = values.associateWith { it.compareMatches(lotto) }
+
+    infix operator fun plus(other: Lottos): Lottos {
+        this.values += other.values
+        return this
+    }
+}
+
+fun List<List<Int>>.toLottos(): Lottos = Lottos(this.map { Lotto(*it.toIntArray()) })
```

Comment:
Why not send a message for the object to work instead of getting the lottos out from the outside(get)? Creating an Object can also be part of a message. 🙂 

ref: [bennadel.com/resources/uploads/2012/ObjectCalisthenics.pdf](https://www.bennadel.com/resources/uploads/2012/ObjectCalisthenics.pdf)

> Rule 4: First class collections
Application of this rule is simple: any class that contains a collection should contain no other
member variables. Each collection gets wrapped in its own class, so now behaviors related to
the collection have a home. You may find that filters become a part of this new class. Also,
your new class can handle activities like joining two groups together or applying a rule to each
element of the group.

Reply:
This comment was resolved on this [comment](https://github.com/next-step/kotlin-lotto/pull/1155#discussion_r2040635403) as well :)